Maple2.Server.Game/PacketHandlers/InstrumentHandler.cs

```diff
@@ -267,6 +285,7 @@ private enum Command : byte {
         }
 
         fieldInstrument = session.Field!.SpawnInstrument(session.Player, metadata);
+        fieldInstrument.StartTick = Environment.TickCount;
```

Comment:
For any tick values that get stored, I've tried to use `TickCount64()` instead. `TickCount()` will wrap around after 25 days system uptime and could cause problems. It's fine to cast the value back to `int` if you need to use it as an `int` since that's equivalent to being truncated, but in this case since you're just finding the delta it won't be a problem.